src/components/Reports/MPGAIncomeExpensesReport/Helper/transformEnums.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  StaffExpenseCategoryEnum,
+  StaffExpensesSubCategoryEnum,
+} from 'src/graphql/types.generated';
+
+export const transformCategory = (value: StaffExpenseCategoryEnum) => {
```

Comment:
Methods that do something similar are called `getLocalized*`. For example: `getLocalizedLikelyToGive` in `src/utils/functions/getLocalizedLikelyToGive.ts`